src/wymeditor/core.js

```diff
@@ -534,7 +519,7 @@ jQuery.extend(WYMeditor, {
         // Path to the main JS files
         wym._options.basePath = wym._options.basePath ||
             WYMeditor._computeBasePath(wym._options.wymPath);
-        // Path to jQuery (for loading in pop-up dialogs)
+        // Path to jQuery
         wym._options.jQueryPath = wym._options.jQueryPath ||
             WYMeditor._computeJqueryPath();
```

Comment:
According to my `git grep`, it's no longer necessary. Let's get rid of it and update the docs.